sw/device/tests/sim_dv/meson.build

```diff
@@ -194,6 +194,34 @@ sw_tests += {
   }
 }
 
+pwrmgr_deep_sleep_all_wake_ups_lib = declare_dependency(
```

Comment:
can you add this target to the (bazel) BUILD file?

Reply:
added